src/interrupt/hw/apic_timer.rs

```diff
@@ -1,15 +1,23 @@
-use crate::interrupt::{apic::local::LOCAL_APIC, InterruptFrame};
-use crate::process::{
-	context::switch_stack,
-	task::{CURRENT, TASK_QUEUE},
+use crate::{
+	interrupt::{apic::local::LOCAL_APIC, InterruptFrame},
+	process::{
+		context::{cpu_context, switch_stack},
+		task::{CURRENT, TASK_QUEUE},
+	},
+	sync::cpu_local::CpuLocal,
 };
 
 use kfs_macro::context;
 
 #[context(hw_irq)]
 pub unsafe extern "C" fn handle_timer_impl(_frame: InterruptFrame) {
+	*JIFFIES.get_mut() += 1;
```

Comment:
Local APIC로 JIFFIES를 갱신하기에는 타이머 주기가 일정하지 않아서
나중에 PIT나 HPET같은걸로 수정하면 좋을 것 같습니다

Reply:
Jiffies 값 의미가 타이머 인터럽트가 몇번 발생했는지에 대한 거지 않나요?

Reply:
I fixed it.
In timer interrupt, context switching is done by manually(not macro) because it is the easiest way.
If I find more compact codes, I will fix it later.